docs/make.jl

```diff
@@ -74,4 +106,5 @@ makedocs(
 
 deploydocs(
     repo   = "github.com/jump-dev/JuMP.jl.git",
+    push_preview = true,
```

Comment:
I see that the doc of this PR is now available at https://jump.dev/JuMP.jl/previews/PR2393/, that's useful